src/blocks/media-block/ImageBlock.new.test.tsx

```
@@ -0,0 +1,463 @@
+  getAttribute() { return null; }
+  setAttribute() {}
+  removeAttribute() {}
+  hasAttribute() { return false; }
+  getBoundingClientRect() { return { width: this.width, height: this.height, top: 0, left: 0, right: this.width, bottom: this.height, x: 0, y: 0, toJSON: () => ({}) }; }
+}
+
+// Extend global types
+declare global {
+  // For test environment flag
+  // eslint-disable-next-line no-var
+  var __VITEST__: boolean | undefined;
```

Biome (1.9.4)

[error] Shouldn't redeclare '__VITEST__'. Consider to delete it or rename it.

'__VITEST__' is defined here:


(lint/suspicious/noRedeclare)